app/views/microcredit/index.html.erb

```diff
@@ -58,9 +58,9 @@ input.button, a.button { background-color: <%=@brand_config["color2"]%> !importa
 <% end %>
 
 <!-- % if @microcredits_mailing.length > 0 or @upcoming_microcredits_mailing.length > 0 or @finished_microcredits_mailing.length > 0 % -->
-  <!--h2><!--%= t 'microcredit.mailing.title', name: @brand_config["name"]%></h2-->
-  <!--p><!--%= t 'microcredit.mailing.help_us', name: @brand_config["name"]%--> <!--%= link_to t('microcredit.common.more_info'), (@brand_config["mailing_info_link"] || microcredits_info_mailing_path), target: "_top" %></--p>
-<!--% end %-->
+  <!--h2>< !--%= t 'microcredit.mailing.title', name: @brand_config["name"]%></h2-->
```

Comment:
I'd remove all this code

Reply:
This block is to allow microcredits of mailing type, I would keep it to allow other persons to use it only uncommenting these lines.